application/__init__.py

```diff
@@ -17,13 +17,21 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
-
+import logging.config
+import yaml
 import os
 import sys
 
 from flask import Flask, Blueprint
 from werkzeug.serving import WSGIRequestHandler
 
+with open('logging_default_config.yaml', 'rt') as config_file:
+    config = yaml.safe_load(config_file.read())
+
+logging.config.dictConfig(config)
+logger = logging.getLogger(__file__)
+logger.debug("successfully set up logger")
```

Comment:
1. In the case the logger isn't set up properly, the log level remains at `WARN` and this debug log is not printed at all.
2. Let's print out the parsed config to better tracability.
```suggestion
logger.warning(f"Logging setup is done with config={config}")
```